server/models/group.js

```diff
@@ -19,7 +19,10 @@ const GroupSchema = new mongoose.Schema({
     unique: true
   },
   users: {
-    type: [mongoose.Schema.Types.ObjectId],
+    type: [{
```

Comment:
Because the required field is in the object, we need it to be nested in the object.